attempt to remove plugin from docs (#1219)
minio/docs: source/operations/install-deploy-manage/deploy-minio-tenant.rst

This documentation assumes familiarity with all referenced Kubernetes concepts, utilities, and procedures. 
While this documentation *may* provide guidance for configuring or deploying Kubernetes-related resources on a best-effort basis, it is not a replacement for the official :kube-docs:`Kubernetes Documentation <>`.

> 

Reviewer: Why four blank lines?

Prerequisites
-------------

MinIO Kubernetes Operator
~~~~~~~~~~~~~~~~~~~~~~~~~

The procedures on this page *requires* a valid installation of the MinIO
Kubernetes Operator and assumes the local host has a matching installation of
the MinIO Kubernetes Operator. This procedure assumes the latest stable Operator, version |operator-version-stable|.

See :ref:`deploy-operator-kubernetes` for complete documentation on deploying the MinIO Operator.

.. cond:: k8s and not (openshift or eks or gke or aks)

   Kubernetes Version 1.21.0
   ~~~~~~~~~~~~~~~~~~~~~~~~~

>    MinIO Operator requires Kubernetes 1.21.0 and later.

Reviewer: Not sure how you have all versions of Kubernetes from 1.21.0 on installed at the same time. 😆 

```suggestion
   MinIO Operator requires Kubernetes 1.21.0 or later.
```

Author: _ALL THE VERSIONS!!_ 🤣

>    The Kubernetes infrastructure *and* the ``kubectl`` CLI tool must have the same version of 1.21.0+.

Reviewer: ```suggestion
   The Kubernetes infrastructure *and* the ``kubectl`` CLI tool must be the same version.
   Upgrade ``kubectl`` to the same version as the Kubernetes version used on the cluster.
```

   * - Field
     - Description

   * - :guilabel:`MinIO`
     - The container image to use for the MinIO Server. 
       See the `MinIO Quay <https://quay.io/repository/minio/minio>`__ or the `MinIO DockerHub <https://hub.docker.com/r/minio/minio/tags>`__ repositories for a list of valid tags.

   * - :guilabel:`KES Image`
     - The container image to use for MinIO :minio-git:`KES <kes>`.

   * - :guilabel:`Use a private container registry`

>      - If the tenant requires a private container registry, toggle to :guilabel:`ON` and specify the location and credentials for the registry.

Reviewer: ```suggestion
     - If the tenant requires a private container registry, toggle to :guilabel:`ON`, then specify the location and credentials for the private registry.
```

     - Description
   
   * - :guilabel:`Vault`
     - Configure `HashiCorp Vault <https://www.vaultproject.io/>`__ as the external KMS for storing root encryption keys. 
       See :ref:`minio-sse-vault` for guidance on the displayed fields.

   * - :guilabel:`AWS`
     - Configure `AWS Secrets Manager <https://aws.amazon.com/secrets-manager/>`__ as the external KMS for storing root encryption keys. 
       See :ref:`minio-sse-aws` for guidance on the displayed fields.

   * - :guilabel:`Gemalto`
     - Configure `Gemalto (Thales Digital Identity and Security) <https://github.com/minio/kes/wiki/Gemalto-KeySecure/>`__ as the external KMS for storing root encryption keys.

>        See `Thales CipherTrust Manager <https://min.io/docs/kes/integrations/thales-ciphertrust/>`__ for guidance on the displayed fields.

Reviewer: ```suggestion
       See :kes-docs:`Thales CipherTrust Manager <integrations/thales-ciphertrust/>`__ for guidance on the displayed fields.
```

.. _minio-tenant-audit-logging-settings:

.. _create-tenant-deploy-view-tenant:

> 11) Deploy and View the Tenant

Reviewer 2: Renumber to 9 ->
